### PR TITLE
Prepare for v0.4.0 release

### PR DESCRIPTION
**Pull request by dagwieers**
Update changelog.txt and addon.xml for release.

### CODE REVIEW

**Comment by dagwieers** [comment]
Still on our TODO list for the v0.4.0 release:
- Retranslate (provide a small window for translations, subsequent translations go into v0.4.1) #103
- Appeal to create a test-group that covers our test plan, and test!

File: addon.xml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="script.module.inputstreamhelper" version="0.3.5" name="InputStream Helper" provider-name="emilsvennesson">
+<addon id="script.module.inputstreamhelper" version="0.4.0" name="InputStream Helper" provider-name="emilsvennesson">
```
Comment (emilsvennesson) [inline addon.xml]: You should definitely add yourself in ```provider-name```. :-)

Comment (dagwieers) [reply addon.xml]: I will leave that up to you. I am not going to do self-promotion :-)

**Comment by dagwieers** [comment]
We are almost ready to release v0.4.0, please help in testing and translating the master branch!
(It is quite easy, download [this zip file](https://github.com/emilsvennesson/script.module.inputstreamhelper/archive/master.zip) and install the add-on as zip from the Add-on menu)

cc @emilsvennesson @asciidisco @vlmaksime @horstle @pinoelefante @mkreisl @gismo112 @dnicolaas @dawez @WallyCZ @Twilight0 @Rechi @Maven85 @morajonas @ZefQ @maxgalbu @Gaet81 @ivankokan @rickybiscus @mrfixit2001 @JohnPlayerSpecial2018 @AlCalzone @rgainza @brianjmurrell @SinafdeGiraf @seankurth @joendres @NickkoG @flubshi @Acromion1754 @MaxMatti @kaminsky9444 @Lattay @cache-sk @basrieter @nicoolaro @matthuisman @peno64

**Comment by dagwieers** [comment]
(Reminder to update the changelog with the translation updates and credentials before merging this)

**Comment by emilsvennesson** [comment]
@dagwieers Sorry for causing a merge conflict but I felt l /had/ to add you to the authors!

**Comment by dagwieers** [comment]
If someone else wants to attempt to get v0.4.0 released, or rework it into v0.4.1, please step forward.
I have given my best.

**Comment by Twilight0** [comment]
I would like to assist, especially in regards to the addon.xml extension points, but I am really busy these days. If you can wait a few, I will send a PR to fix those.

**Comment by horstle** [comment]
@Twilight0  you might want to have a look at #141 . Extension points are modified there, too.

**Pull request merged**